kotsadm/web/src/components/UploadAirgapBundle.jsx

```diff
@@ -294,11 +293,29 @@ class UploadAirgapBundle extends React.Component {
     }, 1000);
   }
 
+  getSupportBundleCommand = async (slug) => {
+    try {
+      const res = await fetch(`${window.env.API_ENDPOINT}/troubleshoot/app/${slug}/supportbundlecommand`, {
+        method: "GET",
+        headers: {
+          "Authorization": Utilities.getToken(),
+        }
+      });
+      if (!res.ok) {
```

Comment:
we added error handling in UI. 
can you make this like this?
```      if (!res.ok) {
        this.setState({ errorMsg: `Unexpected status code: ${res.status}" });
        return;
      }
        this.setState({ errorMsg: ""  });
} catch(err) {
      console.log(err)
      this.setState({ showErrMsg: err  });
}```

https://github.com/replicatedhq/kots/blob/e2bd48da91a7b682a3d496017fc8a233425d70a8/kotsadm/web/src/components/troubleshoot/SupportBundleList.jsx#L40